configs/samples/pjsip.conf.sample

```diff
@@ -998,6 +998,10 @@
                 ; AOC updates can be sent using the AOCMessage AMI action or come
                 ; from PRI channels.
                 ; (default: no)
+;
+; tenantid =
+		; Sets the tenant ID for this endpoint. Channels will have this
```

Comment:
Needed to clarify
- what channel variable will have this value
- what can be changed via dialplan... endpoint option tenantid or channel variable tenantid.
For example:
"The channel variable tenantid will have this value  set on channel creation, and the channel variable tenantid can be changed later via dialplan."